frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -48,27 +50,33 @@ type WithIndex<T> = T & {
   index: number;
 };
 
-export function ExpressionEditorSuggestions({
-  query,
-  stageIndex,
-  suggestions = [],
-  onSuggestionMouseDown,
-  open,
-  highlightedIndex,
-  onHighlightSuggestion,
-  children,
-}: {
-  query: Lib.Query;
-  stageIndex: number;
-  suggestions?: (Suggestion | SuggestionFooter | SuggestionShortcut)[];
-  onSuggestionMouseDown: (index: number) => void;
-  open: boolean;
-  highlightedIndex: number;
-  onHighlightSuggestion: (index: number) => void;
-  children: ReactNode;
-}) {
-  const ref = useRef(null);
-
+export const ExpressionEditorSuggestions = forwardRef<
+  HTMLDivElement,
```

Comment:
The ref is going on a `ul` element (`ExpressionList`).

```suggestion
  HTMLUListElement,
```